ocaml/sdk-gen/go/gen_go_helper.ml

```diff
@@ -41,16 +41,25 @@ let acronyms =
 
 let is_acronym word = StringSet.mem word acronyms
 
-let snake_to_camel (s : string) : string =
+let amend_acronym = function
+  | s when is_acronym s ->
+      String.uppercase_ascii s
+  | s ->
+      String.capitalize_ascii s
+
+let split_str s =
   Astring.String.cuts ~sep:"_" s
   |> List.concat_map (fun s -> Astring.String.cuts ~sep:"-" s)
-  |> List.map (function
-       | s when is_acronym s ->
-           String.uppercase_ascii s
-       | s ->
-           String.capitalize_ascii s
-       )
-  |> String.concat ""
+
+let snake_to_camel (s : string) : string =
+  split_str s |> List.map amend_acronym |> String.concat ""
+
+let snake_to_camel_for_name_internal (s : string) : string =
```

Comment:
How about `let snake_to_camel ?(internal=false) s`?

Reply:
And please add some unit tests for this change.

Reply:
> How about `let snake_to_camel ?(internal=false) s`?

Nice.